client/v3/snapshot/v3_snapshot.go

```diff
@@ -36,13 +36,81 @@ func hasChecksum(n int64) bool {
 	return (n % 512) == sha256.Size
 }
 
+// SaveWithVersion fetches snapshot from remote etcd server, saves data
+// to target path and returns server version. If the context "ctx" is canceled or timed out,
+// snapshot save stream will error out (e.g. context.Canceled,
+// context.DeadlineExceeded). Make sure to specify only one endpoint
+// in client configuration. Snapshot API must be requested to a
+// selected node, and saved snapshot is the point-in-time state of
+// the selected node.
+// Supported since etcd 3.6.
```

Comment:
How it behaves against v3.5 ? 
I assume it  works, just returns ("", err)

Reply:
yes, added comment.